.projenrc.ts

```diff
@@ -26,8 +26,6 @@ const project = new CdklabsConstructLibrary({
     'hpagent',
   ],
   deps: [
-    'aws-cdk-lib@^2.0.0',
-    'constructs@^10.0.5',
```

Comment:
You shouldn't need to specify this. It's controlled by the upstream projen config and the `peerDependencies` it produces.